.readthedocs.yml

```diff
@@ -10,7 +10,7 @@ sphinx:
   configuration: docs/source/conf.py
   # fail_on_warning might generate hard to fix error, in this case it can be
   # disabled but this also means those errors will fail silently, choose wisely.
-  fail_on_warning: false
+  fail_on_warning: true
```

Comment:
Love this !  How many things to fix?

Reply:
Add `-W` to this line in  Makefile https://github.com/Ouranosinc/raven/blob/d0d0a6d4ae652230a9d1ae18e5a95d0d6ab4a4be/docs/Makefile#L23 to also match this `fail_on_warning: true`.

Same as in my original commit https://github.com/Ouranosinc/raven/commit/fa979fff9c38163bd08ff27c973636b4efb664a5

Reply:
> How many things to fix?

The unreferenced notebooks and adding `ipywidgets`. Not much!